security-command-center/snippets/src/test/java/vtwo/AssetSecurityMarksIT.java

```diff
@@ -0,0 +1,145 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import vtwo.assets.AddDeleteSecurityMarks;
+import vtwo.assets.AddSecurityMarksToAssets;
+import vtwo.assets.DeleteAssetsSecurityMarks;
+
+@RunWith(JUnit4.class)
+public class AssetSecurityMarksIT {
+
+  private static final String ORGANIZATION_ID = System.getenv("SCC_PROJECT_ORG_ID");
+  private static String assetId;
+  private static ByteArrayOutputStream stdOut;
+
+  @Rule
+  public final MultipleAttemptsRule multipleAttemptsRule =
+      new MultipleAttemptsRule(3, 120000); // 2 minutes
+
+  // Check if the required environment variables are set.
+  public static void requireEnvVar(String envVarName) {
+    assertThat(System.getenv(envVarName)).isNotEmpty();
+  }
+
+  // Extracts the asset ID from a full resource name.
+  // This regex pattern matches the last segment of the resource name,
+  // which consists of digits after the final forward slash (e.g., "assets/12345").
+  private static String extractAssetId(String assetPath) {
+    // Pattern to match the asset ID at the end of the resource name.
+    Pattern pattern = Pattern.compile("assets/([^/]+)$");
+    Matcher matcher = pattern.matcher(assetPath);
+    if (matcher.find()) {
+      return matcher.group(1);
+    }
+    return assetPath;
+  }
+
+  @BeforeClass
+  public static void setUp() throws IOException, InterruptedException {
+
+    // Validate required environment variables.
+    requireEnvVar("SCC_PROJECT_ORG_ID");
+
+    // Load static_asset.json from resources
+    // Since there are no APIs to create an Asset
+    InputStream inputStream =
+        AssetSecurityMarksIT.class.getClassLoader().getResourceAsStream("static_asset.json");
+
+    if (inputStream == null) {
+      throw new IOException("static_asset.json file not found in resources.");
+    }
+
+    // Convert InputStream to String
+    String jsonContent = new String(inputStream.readAllBytes(), StandardCharsets.UTF_8);
+
+    // Parse JSON (using Gson)
+    JsonObject jsonObject = JsonParser.parseString(jsonContent).getAsJsonObject();
+
+    // Extract assetId from mock data
+    assetId = extractAssetId(jsonObject.get("name").getAsString());
+
+    if (assetId == null || assetId.isEmpty()) {
+      throw new IllegalStateException("Asset ID is missing from static_asset.json");
+    }
+  }
+
+  @Before
+  public void beforeEach() {
+    stdOut = new ByteArrayOutputStream();
+  }
+
+  @After
+  public void afterEach() {
+    stdOut = null;
+    System.setOut(null);
+  }
+
+  @AfterClass
+  public static void cleanUp() {
+    System.setOut(System.out);
+  }
+
+  @Test
+  public void testAddSecurityMarksToAsset() throws IOException {
+    com.google.cloud.securitycenter.v2.SecurityMarks response =
```

Comment:
Although unnecessary here, if you need to refer to a type contained in one of the client libraries, import the type rather than using the fully-qualified name.

Reply:
this topic will be reviewed by code sample maintenance team. until the decision is made, please follow Java specific guidelines and use return statements instead of printing to stdou.